### ISSUE TITLE
Access tokens should be located in .env file or be initialized with CI/CD variables

### ISSUE THREAD

**Issue by dflymegold** [Tiaranso/GameOFTeens2023_PYTHON#1]
Hello, Tiaranso, I found security issue that might be compromized by the man-in-the-middle.

https://github.com/Tiaranso/GameOFTeens2023_PYTHON/blob/f292159c79238009c544ddccfc4fa5942c4e8166/bot.py#L237

However, It's your pet-project I guess, but It's very usefull to know that all credentials keys, security keys should be strored in specific .env file or be initialized during CI/CD that offers by software.

Good point to start:
1. https://habr.com/ru/articles/472674/
2. https://dev.to/jakewitcher/using-env-files-for-environment-variables-in-python-applications-55a1

Basically, It's not a real big deal in pet-project but if we are talking about commercials - enviromental variables are a must.

I wish you good luck in your projects 🥇

**Comment by Tiaranso** [comment]
Yes, we are aware of that. The token that is listed in the code does not exist. We are aware of the dangers of this approach and that secret data should be kept in a separate .env file for better security. 

What I also want to point out is that the code here is not the most structured (not at all). It was written hastily by a team for a hackathon. To all you reading this, I recommend that you not use this code as an example for your own work. 

I want to thank you very much for your comments and say hello from the DOU :) I'm very pleased that you were interested in looking at my GitHub.
